internal/cli/atlas/clusters/delete.go

```diff
@@ -104,7 +104,7 @@ Deleting a cluster also deletes any backup snapshots for that cluster.
 				return err
 			}
 			opts.Entry = args[0]
-			return opts.Prompt()
+			return opts.PromptWithMessage("Confirm your backup settings before terminating your cluster.\nAre you sure you want to terminate %s, this action cannot be undone?")
```

Comment:
can you please do the same for the `atlas deployments delete` command

Reply:
do you have a suggestion for the message since this same copy doesn't apply for local